app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -357,8 +357,13 @@ class UrlInputFragment :
 
     override fun onStart() {
         super.onStart()
-
-        showKeyboard()
+        activity?.let {
```

Comment:
Typically when you use `.let` that implies transformation and returns the result of the lambda (which we're not using here.  A more semantic usage could be 

```
val showFirstRun = activity?.let { Settings.getInstance(it.applicationContext).shouldShowFirstrun() }
if (showFirstRun) {
    return
}

showKeyboard()
```

Or you could use `?.also` 👍